lib/services.rb

```diff
@@ -37,17 +37,11 @@ def self.elasticsearch(cluster: nil, hosts: ENV['ELASTICSEARCH_HOSTS'] || 'http:
   end
 
   def self.statsd_client
-    @statsd_client ||= Statsd.new.tap { |sd| sd.namespace = "govuk.app.search-api" }
+    Statsd.new.tap { |sd| sd.namespace = "govuk.app.search-api" }
```

Comment:
Is instantiating this class every time `self.statsd_client` called going to cause any problems (performance or semantic)?

Reply:
semantic I wouldn't think so. performance-wise I'm not sure really. I'll add it to the cache to be sure